app/javascript/components/records/action-creators.js

```diff
@@ -401,3 +402,33 @@ export const createCaseFromFamilyMember = ({ familyId, familyMemberId }) => ({
     ]
   }
 });
+
+export const fetchLinkIncidentToCaseData = (payload) => {
+  return {
+    type: `cases/${FETCH_LINK_INCIDENT_TO_CASE_DATA}`,
+    api: {
+      path: `${RECORD_PATH.cases}?query=${payload.query}&id_search=${payload.id_search}`
+    }
+  };
+};
+
+export const linkIncidentToCase = ({ recordType, incident_ids = [], case_id }) => {
+  return {
+    type: `${recordType}/LINK_INCIDENT_TO_CASE`,
+    api: {
+      path: `incidents/link_incidents_to_case`,
+      method: "POST",
+      body: { data: { incident_case_id: case_id, incident_ids: incident_ids } },
+      successCallback: {
+        action: ENQUEUE_SNACKBAR,
+        payload: {
+          message: "Linked incident to case",
```

Comment:
This need to be translated, I believe you can use the dot notation i18n path as a string.